extension/cgroupruntimeextension/integration_test.go

```diff
@@ -144,6 +168,71 @@ func TestCgroupV2SudoIntegration(t *testing.T) {
 			// 134217728 * 0.1
 			expectedGoMemLimit: 13421772,
 		},
+		{
+			name:            "AWS ECS 90% the max cgroup memory and 12 GOMAXPROCS",
+			cgroupCpuQuota:  pointerInt64(100000),
```

Comment:
Wdyt of using variable to set the task/container CPU limits? (instead of `expectedGoMaxProcs*1024`)?

Reply:
Discussion continuing here: https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/36920#discussion_r1899713673